subsys/logging/Kconfig

```diff
@@ -295,6 +295,53 @@ config LOG_BACKEND_NATIVE_POSIX
 	help
 	  Enable backend in native_posix
 
+config LOG_BACKEND_NET
+	bool "Enable networking backend"
+	depends on NETWORKING
+	select NET_CONTEXT_NET_PKT_POOL
+	help
+	  Send syslog messages to network server.
+	  See RFC 5424 (syslog protocol) and RFC 5426 (syslog over UDP)
+	  specifications for details.
+
+if LOG_BACKEND_NET
+
+config LOG_BACKEND_NET_SERVER
+	string "Syslog server IP address"
+	help
+	  This can be either IPv4 or IPv6 address.
+	  Server listen UDP port number can be configured here too.
+	  Following syntax is supported:
+	  192.0.2.1:514
+	  192.0.2.42
+	  [2001:db8::1]:514
+	  [2001:db8::2]
+	  2001:db::42
+
+config LOG_BACKEND_NET_MAX_BUF
+	int "How many network buffers to allocate for sending messages"
+	range 3 256
+	default 3
+	help
+	  Each syslog message will occupy one network buffer.
```

Comment:
That's a nice promise, which immediately leads to question "what happens if a logging msg doesn't fit in one net_buf?"

Reply:
Ah, wait, here "networking buffer" might not even mean net_buf.

Reply:
If the syslog message is too long, it is truncated. As this only supports UDP currently, we need to draw the line somewhere.

Reply:
Of course not on the size of net_buf, that would make no sense. I will clarify the text.